conformance/metadata-fhirr4-service/README.md

```diff
@@ -1 +1,61 @@
 # FHIR R4 Metadata Service
+
+## Introduction
+This service provides implementation of FHIR Metadata API. This implements 
+[capabilities](https://www.hl7.org/fhir/http.html#capabilities) interaction, which is used to retrieve capability 
+statement describing the server's current operational functionality by FHIR client applications. 
+
+This FHIR server interaction returns Capability Statement ([CapabilityStatement](http://hl7.org/fhir/StructureDefinition/CapabilityStatement) 
+FHIR resource) that specifies which resource types and interactions are supported by the FHIR server
+
+```Supported FHIR version is 4.0.1.```
+
+
+## [Optional] Deploy in Choreo
+
+WSO2’s Choreo (https://wso2.com/choreo/) is an internal developer platform that redefines how you create digital experiences. Choreo empowers you to seamlessly design, develop, deploy, and govern your cloud native applications, unlocking innovation while reducing time-to-market. You can deploy the healthcare prebuilt services in Choreo as explained below. 
+
+### Prerequisites
+
+If you are signing in to the Choreo Console for the first time, create an organization as follows:
+
+1. Go to https://console.choreo.dev/, and sign in using your preferred method.
+2. Enter a unique organization name. For example, Stark Industries.
+3. Read and accept the privacy policy and terms of use.
+4. Click Create.
+This creates the organization and opens the Project Home page of the default project created for you.
+
+### Steps to Deploy Metadata Prebuilt Service in Choreo
+1. Create Service Component
+    * Fork the pre-built Ballerina services repository (https://github.com/wso2/open-healthcare-prebuilt-services) to your Github organization.
+    * Follow the official documentation to create and configure a service: https://wso2.com/choreo/docs/develop-components/develop-services/develop-a-ballerina-rest-api/#step-1-create-a-service-component.
+
+    * Once the component creation is complete, you will see the component overview page.
+
+2. Configure and Deploy
+
+    Follow the official documentation to deploy the Metadata prebuilt service to your organization https://wso2.com/choreo/docs/develop-components/develop-services/develop-a-ballerina-rest-api/#step-2-build-and-deploy.
+
+    Following configurations needs to be configured in a `Config.toml` or in Choreo configurable editor.
+
+    | Configuration                | Description                                                                                        |
```

Comment:
Shall we have this configurations in a topic so that it can be linked in other places where running locally or deploying in choreo options?

Reply:
Updated in https://github.com/wso2/open-healthcare-prebuilt-services/commit/8b982af37b97ff9cd7f6f9e8a273a8c6a9400010